providers/providerverifier.go

```diff
@@ -132,6 +132,12 @@ func (v *DefaultProviderVerifier) VerifyIDToken(ctx context.Context, idToken []b
 			return fmt.Errorf("failed to get OP public key: %w", err)
 		}
 
+        // Ensure that the algorithm of public key from OpenID Provider matches the algorithm specified in the ID Token
```

Comment:
Linter is angry because tabs instead of spaces

```suggestion
		// Ensure that the algorithm of public key from OpenID Provider matches the algorithm specified in the ID Token
```